pegasus/sites.v3/code.org/public/administrators.haml

```diff
@@ -0,0 +1,191 @@
+---
+title: Code.org District Partner Program
+theme: responsive_full_width
+---
+
+%link{href: 'shared/css/design-system-pegasus.css', rel: 'stylesheet'}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
```

Comment:
The way we implement this is still in flux (re: do we host FontAwesome in our codebase or not) so for now I'm using this implementation but can update it in the future if needed.